.github/workflows/PublishTestResults.yml

```diff
@@ -0,0 +1,54 @@
+# ==============================================================================
+# Authors:
+#   Patrick Lehmann
+#   Unai Martinez-Corral
+#
+# ==============================================================================
+# Copyright 2020-2021 The pyTooling Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ==============================================================================
+name: Publish Unit Test Results
+
+on:
+  workflow_call:
+
+jobs:
+  PublishTestResults:
+    name: 📊 Publish Test Results to Issue Comment
+    runs-on: ubuntu-latest
+    if: always()
+
+    steps:
+      - name: ⏬ Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: artifacts
+
+# TODO: @umarcor do we want to make it optional or just kick it out?
+#      - name: 💬 Comment Unit Test Results in Pull Request
+#        uses: EnricoMi/publish-unit-test-result-action@v1
+#        with:
+#          files: artifacts/**/*.xml
+
+      - name: 📊 Publish Unit Test Results
+        uses: dorny/test-reporter@v1
+        with:
+          name: Unit Test Results
+          path: artifacts/**/*.xml
```

Comment:
The path should be an input with a default value (`artifacts/**/*.xml`). See https://github.com/pyTooling/Actions/blob/main/.github/workflows/StaticTypeCheck.yml#L5-L28. So:

```yml
inputs:
  path:
    description: 'Path(s) to unit test results to be reported.'
    required: false
    default: 'artifacts/**/*.xml'
    type: string
```

Reply:
This pattern depends on a directory use in the download artifacts action.

Maybe the last part (`*.xml`) can be made configurable - especially to exclude non JUnit XML files.

Reply:
The point is that the `path` argument can be a list of paths, and not just one location. See https://github.com/ghdl/ghdl/blob/master/.github/workflows/Test.yml#L55-L63 and https://github.com/dorny/test-reporter#usage:

```yml
    # Coma separated list of paths to test results
    # Supports wildcards via [fast-glob](https://github.com/mrmlnc/fast-glob)
    # All matched result files must be of the same format
    path: ''
```

Hence, users might provide:

```
path: >-
  artifacts/path/to/some/dir/*.xml,
  artifacts/another/location/*.xml
```

Where `path/to/some/dir` and `another/location` will probably be the names of the artifacts that were uploaded `with name:`.

It's ok to document that artifacts are downloaded to subdir `artifacts/` by default, but I would not have `*.xml` only as the parameter. Optionally, we might support the following:

```
path: >-
  path/to/some/dir/*.xml,
  another/location/*.xml
```

by adding one step in this workflow, which prepends `artifacts/` to all of those relative paths, but not to the absolute paths (if provided by the user).

Reply:
Done in https://github.com/pyTooling/Actions/pull/13/commits/774ea7970e32cd1983d1fb57a41e1f43cc769347.